pkg/kubefedctl/enable/enable.go

```diff
@@ -333,6 +349,13 @@ func GenerateTypeConfigForTarget(apiResource metav1.APIResource, enableTypeDirec
 	return typeConfig
 }
 
+func printNameResourceVersion(resource metav1.APIResource) string {
```

Comment:
`s/printNameResourceVersion/qualifiedAPIResourceName/`

`print` is a verb implying that this function will take an action.  A noun (e.g. name) indicating the output is more appropriate.

Reply:
That sounds better. Fixed.